### DISCUSSION TITLE
[教程]使用 AutoHotKey 实现在浏览器外划词

### DISCUSSION THREAD

**Discussion by CheeseOn** [crimx/ext-saladict#1051] [Show and tell]
可以使用AutoHotKey实现在浏览器外划词，这是一款非常强大开源的快捷键软件，可以自定义快捷键及其功能。
这里是一个简单示例脚本，选中需要翻译的内容后按下Ctrl+Shift+C进行查词，Alt+Shift+S为沙拉翻译剪辑版内容快捷键：
^+C::
send ^C
send !+S
return
Ctrl键是^，Alt键是!，Win键是#，Shift键是+。大家可以按自己的想法改，原理是发送Ctrl+C复制，Alt+Shift+S翻译。

**Comment by crimx** [comment]
看起来很不错，感谢分享！
